packages/autocomplete-plugin-query-suggestions/README.md

```diff
@@ -0,0 +1,11 @@
+# @algolia/autocomplete-plugin-query-suggestions
+
+A plugin to add Query Suggestions to Algolia Autocomplete.
+
```

Comment:
add some info here to link to the query suggestions feature?

Reply:
Yep, we'll work on an exhaustive doc with Maria so we'll wait for this content before changing this doc multiple times if you don't mind.